packages/office-ui-fabric-react/src/components/Slider/Slider.types.ts

```diff
@@ -11,7 +11,22 @@ export interface ISlider {
 
   focus: () => void;
 }
-
+/**
+ * {@docCategory Slider}
+ */
+export interface ISliderMarksArrayFormat {
+  /**
+   * This is how the JSON should be formated for the custom marks
```

Comment:
All the comments are placed in wrong places 😄 . Also, I noticed you are using `JSON` to describe this interface but technically this is an array of simple javascript objects. I would advise rewording the documentation and every comment through out the code that mentions JSON.

Reply:
oh I see ok just changed the JSON to JavaScript object and where should I place the comments?

Reply:
For example this line documents the interface so it needs to be above the interface definition and above the `{@docCategory Slider}` tag. Same with the rest of the description. Even the naming of the interface is slightly confusing with the word array in it. Perhaps a simpler `ISliderMarks` would be enough.

As for the JSON wording it's enough to call it object.

Reply:
ok I changed the name and the description moved up